routes/api.js

```diff
@@ -0,0 +1,43 @@
+const { Router } = require('express')
+const router = Router ()
+
+// Auth
+router.post('/auth/signup', require('../controllers/api/auth/signup'))
+router.post('/auth/login', require('../controllers/api/auth/login'))
+router.delete('/auth/logout', require('../controllers/api/auth/logout'))
+
+// Public products
+router.get('/products'), require('../controllers/api/products/index')
+router.get('/products/:id'), require('../controllers/api/products/show')
+
```

Comment:
no need to put "index" below
router.get('/products'), require('../controllers/api/products/index')